test/blank.test.js

```diff
@@ -1,8 +1,10 @@
 class Fixture extends React.Component {
   render () {
     return (
-      <div id='parent'>
-        <div id='child' />
+      <div>
```

Comment:
Why are all these necessary?

Reply:
Enzyme 3 changes the way it wraps HTML when returning from the `render` function. The new version returns a cheerio wrapper with a type of `tag` that IS the parent element (instead of a wrapper with a type `root` that contains the parent element).

For example:
  ```html
<div id='parent'>
    <div id='child' />
<div>
```
With the above code in Enzyme 3, calling `wrapper.find('#parent').length` returns `0` while calling `wrapper.is('#parent')` returns `true`. With Enzyme 2, `wrapper.find('#parent').length` returns `1` and `wrapper.is('#parent')` returns `false`.

Reply:
(this is as a result of a breaking change in cheerio v1)

Reply:
Thanks for explaining, we'll definitely have to document this in our `CHANGELOG.md` to educate people on how to upgrade

Reply:
Crazy 3AM idea (jetlag), can we add the extra `<div />` in chai-enzyme when using `render`?

Reply:
You could wrap it, sure, but then anyone who didn't understand this, and was checking *any other method* besides `.html()`, would have the wrong assertions.

In other words, it's better for `wrapper.is('#parent') === true` and `wrapper.html()` not to contain the parent div, then for the reverse to be true.